src/components/markdown-renderer/replace-components/markmap/markmap-frame.tsx

```diff
@@ -45,17 +45,15 @@ export const MarkmapFrame: React.FC<MarkmapFrameProps> = ({ code }) => {
       return
     }
     const actualContainer = diagramContainer.current
-    Promise.all([import(/* webpackChunkName: "markmap" */ 'markmap-lib'), import(/* webpackChunkName: "markmap" */ 'markmap-view')])
-      .then(([{ transform }, { Markmap }]) => {
-        const svg: SVGSVGElement = document.createElementNS('http://www.w3.org/2000/svg', 'svg')
-        svg.setAttribute('width', '100%')
-        actualContainer.querySelectorAll('svg').forEach(child => child.remove())
-        actualContainer.appendChild(svg)
-        const { root } = transform(code)
-        Markmap.create(svg, {}, root)
-      }).catch(() => {
-        console.error('error while loading markmap')
-      })
+    import('./markmap-loader').then(({ markmapLoader }) => {
```

Comment:
Is it intentional that you drop the extra webpackChunkName here?

Reply:
Yes. Most of the time webpack didn't use it or the naming wasn't useful. It also had no effect on the split. You get way more information when you look at the `analyze` output.